ocs_ci/deployment/deployment.py

```diff
@@ -2885,7 +2885,9 @@ def validate_mirror_peer(self, resource_name):
         # on all participating clusters except HUB
         # We will switch config ctx to Participating clusters
         for cluster in config.clusters:
-            if cluster.MULTICLUSTER["multicluster_index"] in get_all_acm_indexes():
+            if cluster.MULTICLUSTER[
```

Comment:
can we have small util functions which can achieve this ? 
```suggestion
            def is_acm_cluster():
              return if cluster.MULTICLUSTER['multicluster_index'] in get_all_acm_indexes()
```

same for is_recovery_cluster() , so that it improves readability and also reusability